### PR TITLE
Verify Login Page Appears While Reaching RHODS Page

### PR DESCRIPTION
**Pull request by risusing**

### CODE REVIEW

**Comment by risusing** [comment]
Jenkins Job Run : https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/541/
Result : Passed.
yesterday when I tested 400_ods_dashboard.robot file whole so it gave us one unexpected failure for this "Verify Documentation Link HTTP Status Code" test case but now it's working fine.

**Review by bdattoma — APPROVED** [review]
lgtm

File: tests/Tests/400__ods_dashboard/400__ods_dashboard.robot [hunk 1]
```diff
@@ -124,3 +130,13 @@ Verify JupyterHub Card CSS Style
...
     ...    operation=contains
+
+Test Setup For Login Page
+    Set Library Search Order  SeleniumLibrary
+    Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+
+Check OpenShift Login Visible
+    ${result}=  Is OpenShift Login Visible
+    IF  ${result}=='false'
+        FAIL    OpenShift Login Visible
```
Comment (tarukumar) [inline tests/Tests/400__ods_dashboard/400__ods_dashboard.robot:141]: I think message should be **"OpenShift Login Is Not Visible"**

Comment (risusing) [reply tests/Tests/400__ods_dashboard/400__ods_dashboard.robot:141]: Yes. I'll update it.

**Review by tarukumar — APPROVED** [review]
LGTM

**Review by bdattoma — APPROVED** [review]
lgmt

**Pull request merged**